packages/components/src/components/button-tile/button-tile.tsx

```diff
@@ -0,0 +1,182 @@
+import type {
+  BreakpointCustomizable,
+  SelectedAriaAttributes,
+  PropTypes,
+  ButtonAriaAttribute,
+  ButtonType,
+  LinkButtonIconName,
+} from '../../types';
+import type {
+  LinkButtonTileSize,
+  LinkButtonTileWeight,
+  LinkButtonTileAspectRatio,
+  LinkButtonTileAlign,
+} from '../../utils';
+import { Component, Element, h, Listen, Prop } from '@stencil/core';
+import {
+  AllowedTypes,
+  attachComponentCss,
+  attachSlottedCss,
+  BUTTON_TYPES,
+  BUTTON_ARIA_ATTRIBUTES,
+  getPrefixedTagNames,
+  parseJSON,
+  validateProps,
+  LINK_BUTTON_TILE_SIZES,
+  LINK_BUTTON_TILE_WEIGHTS,
+  LINK_BUTTON_TILE_ASPECT_RATIOS,
+  LINK_BUTTON_TILE_ALIGNS,
+  throwIfAlignTopAndNotCompact,
+  isDisabledOrLoading,
+} from '../../utils';
+import { getComponentCss } from './button-tile-styles';
+import { getSlottedCss } from '../../styles/link-button-tile-styles';
+import { getButtonAriaAttributes } from '../button/button-utils';
+import { ButtonProps } from './button-tile-utils';
+
+const propTypes: PropTypes<typeof ButtonTile> = {
```

Comment:
Hmm, now basically everything is duplicated?
I thought we wanted to abstract or reuse a thing or two.

Reply:
Yes, you are right. I will have a look at this :)